### ISSUE TITLE
TextMeasurer.Measure gives ArgumentOutOfRangeException when measing text starting or ending with / slash

### ISSUE THREAD

**Issue by FlatlinerDOA** [SixLabors/Fonts#253]
### Prerequisites

- [x] I have written a descriptive issue title
- [x] I have verified that I am running the latest version of Fonts
- [x] I have verified if the problem exist in both `DEBUG` and `RELEASE` mode
- [x] I have searched [open](https://github.com/SixLabors/Fonts/issues) and [closed](https://github.com/SixLabors/Fonts/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description
Stack Trace:
```
System.ArgumentOutOfRangeException: Index was out of range. Must be non-negative and less than the size of the collection. (Parameter 'index')
   at System.Collections.Generic.List`1.get_Item(Int32 index)
   at SixLabors.Fonts.GlyphSubstitutionCollection.GetGlyphShapingData(Int32 index)
   at SixLabors.Fonts.Tables.AdvancedTypographic.Shapers.DefaultShaper.AssignFeatures(IGlyphShapingCollection collection, Int32 index, Int32 count)
   at SixLabors.Fonts.Tables.AdvancedTypographic.GSubTable.ApplySubstitution(FontMetrics fontMetrics, GlyphSubstitutionCollection collection, KerningMode kerningMode)
   at SixLabors.Fonts.StreamFontMetrics.ApplySubstitution(GlyphSubstitutionCollection collection, KerningMode kerningMode)
   at SixLabors.Fonts.FileFontMetrics.ApplySubstitution(GlyphSubstitutionCollection collection, KerningMode kerningMode)
   at SixLabors.Fonts.TextLayout.DoFontRun(ReadOnlySpan`1 text, TextOptions options, FontMetrics fontMetrics, BidiRun[] bidiRuns, Dictionary`2 bidiMap, GlyphSubstitutionCollection substitutions, GlyphPositioningCollection positionings)
   at SixLabors.Fonts.TextLayout.ProcessText(ReadOnlySpan`1 text, TextOptions options)
   at SixLabors.Fonts.TextLayout.GenerateLayout(ReadOnlySpan`1 text, TextOptions options)
   at SixLabors.Fonts.TextMeasurer.TextMeasurerInt.Measure(ReadOnlySpan`1 text, TextOptions options)
   at SixLabors.Fonts.TextMeasurer.Measure(String text, TextOptions options)
```

### Steps to Reproduce
Sample code
```
    var font = SystemFonts.CreateFont("Arial", 20, FontStyle.Bold);
    var rendererOptions = new TextOptions(font)
    {
        WrappingLength = 50,
        Origin = new Vector2(10, 10)
    };
    var measurement = TextMeasurer.Measure("Ending slash /", rendererOptions);
```

### System Configuration
Any

- Fonts version:
- Other Six Labors packages and versions: N/A
- Environment (Operating system, version and so on): Windows 10
- .NET Framework version: 6.0

**Comment by JimBobSquarePants** [comment]
Duplicate of

https://github.com/SixLabors/Fonts/issues/244

**Comment by FlatlinerDOA** [comment]
@JimBobSquarePants has the fix been deployed to NuGet? 1.0.0-beta16 (latest prerelease) still has the issue.

**Comment by JimBobSquarePants** [comment]
It's on our MyGet feed. Currently  [1.0.0-beta16.11](https://www.myget.org/feed/sixlabors/package/nuget/SixLabors.Fonts/1.0.0-beta16.11)
